.idea/misc.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (Introduction-to-Machine-Learning)" project-jdk-type="Python SDK" />
+  <component name="PyCharmProfessionalAdvertiser">
```

Comment:
The PyCharmProfessionalAdvertiser component is not necessary and should be removed.